addon/mixins/pikaday.js

```diff
@@ -239,5 +250,17 @@ export default Mixin.create({
 
   _updateOptions() {
     this.get('pikaday').config(this.get('_options'));
+  },
+
+  verifyInvalidDate(element) {
+    if (this.get('clearInvalidDate')) {
+      var validDate = moment(element.val(), this.get('format'),true ).isValid();
```

Comment:
Can you add a comment here for a "future traveller" about that that third `true` argument to `moment()` does? It would be helpful for whoever is reviewing this code in the future.